cmd/tailscaled/tailscaled.go

```diff
@@ -540,24 +540,24 @@ func getLocalBackend(ctx context.Context, logf logger.Logf, logid string) (_ *ip
 	return lb, nil
 }
 
-func createEngine(logf logger.Logf, linkMon *monitor.Mon, dialer *tsdial.Dialer) (e wgengine.Engine, useNetstack bool, err error) {
+func createEngine(logf logger.Logf, linkMon *monitor.Mon, dialer *tsdial.Dialer) (e wgengine.Engine, userspaceNetworking bool, err error) {
 	if args.tunname == "" {
 		return nil, false, errors.New("no --tun value specified")
 	}
 	var errs []error
 	for _, name := range strings.Split(args.tunname, ",") {
 		logf("wgengine.NewUserspaceEngine(tun %q) ...", name)
-		e, useNetstack, err = tryEngine(logf, linkMon, dialer, name)
+		e, userspaceNetworking, err = tryEngine(logf, linkMon, dialer, name)
 		if err == nil {
-			return e, useNetstack, nil
+			return e, userspaceNetworking, nil
 		}
 		logf("wgengine.NewUserspaceEngine(tun %q) error: %v", name, err)
 		errs = append(errs, err)
 	}
 	return nil, false, multierr.New(errs...)
 }
 
-func shouldWrapNetstack() bool {
+func handleSubnetsInNetstack() bool {
```

Comment:
docs then while you're here and apparently caring about such details?

Reply:
Done.